drizzlepac/haputils/poller_utils.py

```diff
@@ -768,9 +771,30 @@ def build_poller_table(input, log_level, poller_type='svm'):
             # This code interprets that as False since it is NOT new, so the code
             # inverts the meaning from the pipeline poller file.
             if poller_type == 'mvm':
+                # Translate new format back to old format ("NEW" -> 0 and "OLD" -> 1)
+                poller_table_mapping = {"NEW": 0, "OLD": 1}
+                for tbl_ctr in range(0, len(input_table)):
+                    if input_table[tbl_ctr]['skycell_new'] in ["OLD", "NEW"]:
+                        input_table[tbl_ctr]['skycell_new'] = poller_table_mapping[input_table[tbl_ctr]['skycell_new']]
                 input_table['skycell_new'] = [int(not BOOL_STR_DICT[str(val).upper()]) for val in input_table['skycell_new']]
             is_poller_file = True
 
+            # Check that each file listed in the poller file exists in the current working directory. If a
+            # file is missing, copy it over from the path specified in the poller file. Failing that, raise
+            # an exception and exit.
+            for table_line in input_table:
+                if os.path.exists(table_line['filename']):
+                    log.info("Input image {} found in current working directory.".format(table_line['filename']))
+                elif os.path.exists(table_line['pathname']):
+                    log.info("Input image {} not found in current working directory. However, it was found in the path specified in the poller file.".format(table_line['filename']))
+                    shutil.copy(table_line['pathname'], os.getcwd())
+                    log.info("Input image {} copied to current working directory.".format(table_line['pathname']))
+                else:
+                    log.error("Input image {} not found in current working directory.".format(table_line['filename']))
+                    log.error("Input image {} not found.".format(table_line['pathname']))
```

Comment:
This log message should be a little more specific, as opposed to being confusingly similar to the previous line.  Something like 'Archived input image {} not found' might be a little clearer.

Reply:
Issue resolved in commit [79ca373](https://github.com/spacetelescope/drizzlepac/pull/874/commits/79ca373b5297551ecebfff51866b0554c551808a)